src/asim/configs/common/constants.yaml

```diff
@@ -220,10 +187,6 @@ xfers_drv_multiplier: 20
 drvtrn_distpen_0_multiplier: 270
 drvtrn_distpen_max: 15
 density_index_multiplier: -0.2
-#    joint_sr2_ASC_no_auto: 0
-#    joint_sr2_ASC_auto_deficient: 0
-#    joint_sr2_ASC_auto_sufficient: 0
-#    joint_drive_transit_ASC_no_auto: 0
 cost_share_s2: 0.57
```

Comment:
cost_share_s2 and cost_share_s2 values differ between common and crossborder constants.yaml. Need to update crossborder trip_mode_choice_annotate_trips_preprocessor.csv and write_trip_matrices_annotate_trips_preprocessor.csv to divide by this value rather than multiply (consistent with resident configs)

Reply:
Good catch! I have committed fixes to this (as well as the other one with the same issue). It will change the results slightly but I don't think it will have any significant impacts.